### ISSUE TITLE
Getting error entityManagerFactory not found

### ISSUE THREAD

**Issue by IOgraph** [spring-guides/gs-accessing-data-jpa#29]
***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of method demo in hello.Application required a bean named 'entityManagerFactory' that could not be found.

Action:

Consider defining a bean named 'entityManagerFactory' in your configuration.

**Comment by gregturn** [comment]
Try cleaning your ~/.m2 directory and running it again.

**Comment by dmarquezbh** [comment]
@gregturn saved my life!

**Comment by nickleus** [comment]
[ERROR] Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:2.0.5.RELEASE:run (default-cli) on project gs-accessing-data-jpa: An exception occurred while running. null: Invoca
tionTargetException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocatio
n of init method failed; nested exception is java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException: javax.xml.bind.JAXBException

i ran these, but it didn't help:
mvn clean install
mvn dependency:purge-local-repository clean install

**Comment by mhuisman** [comment]
the reference `pom.xml` needs to be updated: 

```include::https://raw.githubusercontent.com/spring-guides/{project_id}/master/initial/pom.xml[]```

I added the following:

```
        <dependency>
            <groupId>javax.xml.bind</groupId>
            <artifactId>jaxb-api</artifactId>
            <version>2.3.1</version>
        </dependency>
```

**Comment by mhuisman** [comment]
Fixed in #36
